content/en/docs/apidocs-mxsdk/apidocs/design-properties.md

```diff
@@ -237,7 +542,149 @@ Here is an example of a **Toggle** property that was renamed from a **Dropdown**
 
 The design property above is a replacement for the removed option **Styling option 3** of **My Dropdown Property** and will be set to **Yes** if that option was selected. The value of **My Dropdown Property** will then be set to empty if that design property still exists.
 
-## 7 Read More
+### 6.2 Renaming multiple Toggle properties to a multi-select ToggleButtonGroup Property
+
+You can rename several **Toggle** properties to a multi-select **ToggleButtonGroup** property. To do this, use the `oldNames` field of an option to refer to the name of the corresponding **Toggle**. Make sure to set the `multiSelect` field of the **ToggleButtonGroup** to **true**.
+
+Here is an example of a **ToggleButtonGroup** property that was renamed from several **Toggle** properties:
+
+```js
+{
+    "name": "Font styling",
+    "type": "ToggleButtonGroup",
+    "description": "Change font styling",
+    "multiSelect": true,
+    "options": [
+        {
+            "name": "Bold",
+            "text": "bold",
+            "oldNames": ["Bold text"],
+            "class": "text-bold"
+        },
+        {
+            "name": "Italic",
+            "text": "italic",
+            "oldNames": ["Italic text"],
+            "class": "text-italic"
+        },
+        {
+            "name": "Underline",
+            "text": "underline",
+            "oldNames": ["Underline text"],
+            "class": "text-underline"
+        }
+    ]
+}
+```
+
+The design property above is a replacement for the removed properties **Bold text**, **Italic text** and **Underline text**. An option will be turned on if that **Toggle** was previously turned on. The value of **Toggle** will then be turned off if that design property still exists.
+
+### 6.3 Renaming a Dropdown option to a Spacing option
+
+You can rename options of a **Dropdown**, **Colorpicker** or **ToggleButtonGroup** property to an option of a **Spacing** property. In this case, the old name of the renamed options consists of the **Dropdown**, **Colorpicker** or **ToggleButtonGroup** property's name and the option's name separated by two colons. It is not required for the original property to still exist — it may have been removed entirely.
```

Comment:
I think we should avoid confusion here and just settle for the dropdown for now, even though the alternatives are technically possible